watcher/exts/snaxcog.py

```diff
@@ -289,21 +370,36 @@ async def _lucrative_batters(self, ctx, count: int = 3):
                       "between seasons!\n\n"
         else:
             message = ""
-        for player in luc_list[:count]:
-            name = player[1]["player"]["fullName"]
-            player_id = player[1]["player"]["id"]
-            shorthand = player[1]["team"]["team_abbreviation"]
-            entry = f"[{name}]({'https://www.blaseball.com/player/' + player_id}) "
-            stats = player[0]
-            message += f"Coins earned this season from {entry} ({shorthand}):\n"
-            total = stats['hits'] + stats['home_runs'] + stats['stolen_bases']
-            message += f"Total: {total:,} - Hits: {stats['hits']:,} " \
-                       f"HRs: {stats['home_runs']:,} SBs: {stats['stolen_bases']:,}\n\n"
-        embed = discord.Embed(colour=discord.Colour.green(),
-                              title=title, description=message)
-        if not snax_set:
-            embed.set_footer(text="You'll get better results if you set your snaxfolio!")
-        await ctx.send(embed=embed)
+        if output_format == 'csv':
+            filename = f"lucrative_batters.csv"
+            fieldnames = ["player", "team", "total", "hits", "home_runs", "stolen_bases"]
+            with open(os.path.join('data', 'tmp', filename), 'w', newline='') as csvfile:
+                writer = csv.DictWriter(csvfile, delimiter=',', fieldnames=fieldnames)
+                writer.writeheader()
+                for player in luc_list:
+                    stats = player[0]
+                    writer.writerow({"player": player[1]["player"]["fullName"], "team": player[1]["team"]["full_name"], "total": stats['hits'] + stats['home_runs'] + stats['stolen_bases'], "hits": stats['hits'], "home_runs": stats['home_runs'], "stolen_bases": stats['stolen_bases']})
```

Comment:
this should probably list hits minus homeruns as that's how the payout works. i don't *think* it's save that way prior to this but double check.